terraform/deployments/create-or-update/main.tf

```diff
@@ -42,6 +42,38 @@ resource "azurerm_nginx_deployment" "example" {
   tags = var.tags
 }
 
+resource "azurerm_nginx_configuration" "aniruddh-example-config" {
```

Comment:
can we change the resource name to not have your user_id(aniruddh)

Reply:
Thanks @arpith-f5, changed.